doc/cygnus-common/backends_catalogue/sql_backend.md

```diff
@@ -0,0 +1,16 @@
+# SQL backend
```

Comment:
Is there any ToC or index document in which this new documentation file should be linked?

In addition, maybe (not sure) it should be added to mkdocs.yml file.

Reply:
Maybe the file modify is this one: doc/cygnus-common/backends_catalogue/README.md

In this file (and laso mkdocs.yml) it is a matter not only of adding the new .md, but also to remove the ones removed.

Reply:
Fixed in [f880887](https://github.com/telefonicaid/fiware-cygnus/pull/1842/commits/f880887ecdb4c3d54ae9b1bcc0e3fd2037202cbb)